packages/gatsby-source-contentful/src/fragments.js

```diff
@@ -1,6 +1,22 @@
 import { graphql } from "gatsby"
 
-export const contentfulAssetFixed = graphql`
+/**
+ * The simplest set of fields for fixed assets
+ * @type {Fragment}
+ * @example
+ * myContentfulAssetField {
+ *   fixed {
+ *     ...GatsbyContentfulFixed
+ *     # ^ identical to using the following fields:
+ *     # base64
+ *     # width
+ *     # height
+ *     # src
+ *     # srcSet
+ *   }
+ * }
+ */
+export const GatsbyContentfulFixed = graphql`
```

Comment:
I don't think renaming these causes problems but it made the name of the fragment usable with JSDoc and the doc transformer so that the name of the const matches the fragment name to prevent confusion.

Reply:
Similar to page queries - export names are being discarded, and Gatsby only cares about fragment name(s) inside graphql tagged template literal. So that won't break anything and it's awesome if it will enable nicer docs!